geoh5py/io/h5_reader.py

```diff
@@ -87,6 +87,9 @@ def fetch_attributes(
             for key, value in entity.attrs.items():
                 attributes["entity"][key] = value
 
+            if "Type" not in entity:
```

Comment:
In the writer, you are creating a group called 'Types', but here it is just 'Type'.  Is this correct?

Reply:
hmm.. Based on the tests, I'm guessing that this is correct, but I'll leave the comment in here so you can explain to me what's going on!!

Reply:
That's just geoh5 naming convention. "Types" is a group containing the "Type" of all objects... not my choice

Reply:
gotcha